src/test/java/org/jabref/logic/search/query/SearchQueryLuceneConversionTest.java

```diff
@@ -10,64 +12,55 @@
 
 class SearchQueryLuceneConversionTest {
```

Comment:
Is this really lucene? Isn't it Postgres?

Reply:
> Is this really lucene? Isn't it Postgres?

SQL queries tested in [`SearchQuerySQLConversionTest.java`](https://github.com/JabRef/jabref/blob/postgresql/src/test/java/org/jabref/logic/search/query/SearchQuerySQLConversionTest.java).

 Lucene is used for searching in linked files.